examples/graphql-cache-invalidation/graphql_cache_invalidation.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - Cache invalidation
+
+The Ballerina `graphql` module provides functionality for cache invalidation. The `invalidate()` and `invalidateAll()` APIs in the `graphql:Context` can be used to invalidate caches in a `graphql:Service`. To invalidate a specific cache in a `resource` method, the `invalidate()` API requires to specify the name of the resource method. On the other hand, the `invalidateAll()` API invalidates all caches within the service. Additionally, the `invalidate()` API supports the cache invalidation of specific subfields, as exemplified by the syntax `invalidate("field.sub-field")`. Conversely, the `invalidateAll()` API invalidates all caches within the `graphql:Service`.
```

Comment:
This still isn't accurate. :( The `invalidate` method is for invalidating the cache of a specific field. It's not an additional feature, it is the feature. 

```suggestion
The Ballerina `graphql` module provides functionality for cache invalidation. The `invalidate()` and `invalidateAll()` APIs in the `graphql:Context` can be used to invalidate caches in a `graphql:Service`. The `invalidate()` API supports the cache invalidation of a specific field by providing the full path of the field separated by a full stop(`.`). For example, `invalidate("field.subfield.anotherSubfield")`. Conversely, the `invalidateAll()` API invalidates all caches within the `graphql:Service`.. On the other hand, the `invalidateAll()` API invalidates all caches within the service.
```